### ISSUE TITLE
Usability improvements for onionshare-gui

### ISSUE THREAD

**Issue by ageis** [micahflee/onionshare#190]
The "Start Server" and "Stop server" language could be made more friendly. Does the user need to know that a server is being started? How about just "Share File" or something similar?

Also, since it takes a while to spin up the hidden service and compress the files, there should ideally be a progress bar for both functions.

**Comment by micahflee** [comment]
There can't be a progress bar for the waiting for hidden service because there's no way to estimate how long it will take. Basically it continues to try to make requests in a loop, making a new one after the old one times out, until one of them succeeds. So it could be after the 1st try, it could be after the 5th (but it's normally after around the 3rd). Maybe it would be more helpful if the GUI explained which try it's on.

A progress bar for zipping might more doable though. I opened a separate issue for this: #191.

And the "Start Server"/"Stop Server" language is easy to change. I opened a separate issue for that too: #192
